src/Coalesce.React/entity-search/src/results.js

```diff
@@ -98,7 +99,7 @@ export class SearchResults extends React.Component {
     const { tabledata, columns } = this.state;
 
     // Derive Headers from columns
-    var headers = columns.filter(column => column.show !== false && column.width !== 34).map((item) => item.accessor);
+    var headers = columns.filter(column => column.show !== false && column.width !== COLUMN_BUTTON_WIDTH).map((item) => item.accessor);
 
```

Comment:
This could maybe user a comment.  Testing on the width of a column does not seem reliable.  Another column could have that width.

Reply:
This is looking at the column definition not the actual width as rendered. Therefore changing column widths would have no affect on this check. Only way another column could have the same width is if someone defined it that way in the javascript. The reason it broke was this value was referenced in two locations and got out of sync. Replacing with a const mitigates the chances of this breaking in the future. We could instead add a flag but it would essentially be the same thing.